test/unit/util/locking/RedisResourceLocker.test.ts

```diff
@@ -0,0 +1,65 @@
+import { InternalServerError } from '../../../../src/util/errors/InternalServerError';
+
+describe('A RedisResourceLocker', (): void => {
```

Comment:
You're not testing `RedisResourceLocker` here?

You should use a _real_ `RedisResourceLocker` but mock `redis`, like we do here for SPARQL endpoints: https://github.com/solid/community-server/blob/12ace1b556fcdaccc2bbdfcfe22197b00e0a16be/test/unit/storage/accessors/SparqlDataAccessor.test.ts#L21

and

https://github.com/solid/community-server/blob/12ace1b556fcdaccc2bbdfcfe22197b00e0a16be/test/unit/storage/accessors/SparqlDataAccessor.test.ts#L62-L65